frontend/src/composables/useDrawer.ts

```diff
@@ -94,14 +94,18 @@ export const isItemOnTop = (
 		return false
 
 	if (
-		(item.entityTypeName === 'block' ||
-			item.entityTypeName === 'transaction') &&
+		item.entityTypeName === 'transaction' &&
 		item.entityTypeName === currentTopItem.value.entityTypeName
 	)
 		return !!(
 			(item.hash && item.hash === currentTopItem.value.hash) ||
 			(item.id && item.id === currentTopItem.value.id)
 		)
+	if (
+		item.entityTypeName === 'block' &&
+		item.entityTypeName === currentTopItem.value.entityTypeName
+	)
+		return !!(item.hash && item.hash === currentTopItem.value.hash)
```

Comment:
Since `hash` is always defined:
```suggestion
		return !!(item.hash === currentTopItem.value.hash)
```

Reply:
This is not true. Item is a generic item working on all types of events.